src/main/java/n3/arrays/ArrayTask8.java

```diff
@@ -0,0 +1,25 @@
+package n3.arrays;
+/*
+- написати програму яка рахує середнє значення елементів масиву значення яких є в проміжку [10, 25] та є кратними 3
+ */
+
+public class ArrayTask8 {
+    public static void main(String[] args) {
+        int [] arr = { -5, 0, 1, 2, 3, 5, 28, 6, 8, 11, 56, 87, 54, 34, 21, 9, 80, 81, 90, -100};
+        int avg = 0;
```

Comment:
The same problem here.
Declare variables as late as possible.

Reply:
fixed